### PR TITLE
Unused Traitor Items: Door Charges

### PR DESCRIPTION
**Pull request by Iamgoofball**
Larger AOE, more devestation.

### CODE REVIEW

**Comment by kevinz000** [comment]
:-1:

**Comment by Archie700** [comment]
>make an already deadly weapon even more deadly

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -956,11 +956,11 @@ var/list/airlock_overlays = list()
 		charge.ex_act(1)
 		detonated = 1
 		charge = null
-		for(var/mob/living/carbon/human/H in orange(1,src))
+		for(var/mob/living/carbon/human/H in orange(2,src))
 			H.Paralyse(8)
-			H.adjust_fire_stacks(1)
+			H.adjust_fire_stacks(20)
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: Oh boy an instant kill for everyone near the door if they don't get medical help that sounds fun

**Comment by MMMiracles** [comment]
I mean considering the outcome of using this extremely situational item was either insta-crit or slight bruising, a buff is needed, but I don't know if this is the right way to do it.

Make it blow up both sides of the door instead of the side it was placed on so there isn't a 50/50 chance it gets completely wasted because someone went through from the other side.

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/machinery/doors/airlock.dm]: 20?!!

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -956,11 +956,11 @@ var/list/airlock_overlays = list()
...
 			H.IgniteMob() //Guaranteed knockout and ignition for nearby people
-			H.apply_damage(20, BRUTE, "chest")
+			H.apply_damage(40, BRUTE, "chest")
```
Comment (Xhuis) [inline code/game/machinery/doors/airlock.dm]: I'm actually okay with this, seeing as it's designed to destroy a reinforced airlock and you're standing right next to it.

**Comment by optimumtact** [comment]
It's pretty situational that this ever hits anybody, so I'm okay with it

**Comment by KorPhaeron** [comment]
invisible instant kill would probably be unpleasant though

**Comment by MMMiracles** [comment]
It's literally supposed to be rigging the front door if a department and yelling "HELP SCIENCE" someone like the HOS or department head barges in to explode.

If It can't even reliably do what it's meant for then it's a waste of TC.

**Comment by optimumtact** [comment]
Someone will nerf it if it's too powerful

**Pull request merged**